webofneeds/won-bot/pom.xml

```diff
@@ -10,7 +10,6 @@
 	</parent>
 	<artifactId>won-bot</artifactId>
 	<packaging>jar</packaging>
-	<version>0.4-SNAPSHOT</version>
```

Comment:
I think we still need this version as long as the bot project lives inside our parent project